mxtools/flyer.py

```diff
@@ -113,13 +124,34 @@ def collect_asset_docs(self):
         if not os.path.isfile(self._master_file):
             raise RuntimeError(f"File {self._master_file} does not exist")
 
-        for i in [0]:
-            datum_id = "{}/{}".format(resource_uid, i)
-            self._datum_ids.append(datum_id)
+        # resource = {
+        #     "resource_id": "RES",
+        #     "resource_kwargs": {},  # this goes to __init__
+        #     "spec": "AD-EIGER-MX",
+        #     ...: ...,
+        # }
+        # datum = {
+        #     "datum_id": "a",
+        #     "datum_kwargs": {"data_key": "data"},  # this goes to __call__
+        #     "resource": "RES",
+        #     ...: ...,
+        # }
+        # datum = {
+        #     "datum_id": "b",
+        #     "datum_kwargs": {"data_key": "omega"},
+        #     "resource": "RES",
+        #     ...: ...,
+        # }
+
+        # event = {...: ..., "data": {"eiger_img": "a", "omega": "b"}}
+
```

Comment:
there are a couple of areas of commented-out code. are were going to find these useful enough in the long term to keep them?